internal/util/ctxutil/ctxutil.go

```diff
@@ -74,7 +74,9 @@ func DurationWithJitter(cap time.Duration, retry int64) time.Duration {
 
 	maxMilliseconds := float64(base.Milliseconds()) * math.Pow(2, float64(retry))
 	capMilliseconds := float64(cap.Milliseconds())
-	lowestValue := int64(math.Min(capMilliseconds, maxMilliseconds))
 
-	return time.Duration(rand.Int63n(lowestValue)) * time.Millisecond
+	lowestValue := int64(math.Min(capMilliseconds, maxMilliseconds)) - 100
+
+	// Math/rand is good enough because we don't need the randomness to be cryptographically secure.
+	return time.Duration(rand.Int63n(lowestValue)+100) * time.Millisecond
```

Comment:
I think that would panic with `cap = 50 * time.Millisecond`, for example. Please add a test

Reply:
Should we add a comment that provided cap cannot be smaller than `100ms`, or should we for example decrease min to 0ms if the cap is smaller than 100ms?

Reply:
Also, do we even need minimum jitter?

Reply:
For now I've updated description of function to mention expected parameters, added custom panic message similarly to the one that we already have for negative `retry`, and added test case that validates if it actually panics.